contrib/seeds/README.md

```diff
@@ -8,21 +8,11 @@ and remove old versions as necessary (at a minimum when GetDesirableServiceFlags
 changes its default return value, as those are the services which seeds are added
 to addrman with).
 
-The seeds compiled into the release are created from sipa's DNS seed data, like this:
+The seeds compiled into the release are created from sipa's DNS seed and AS map
+data, like this:
 
     curl https://bitcoin.sipa.be/seeds.txt.gz | gzip -dc > seeds_main.txt
-    python3 makeseeds.py < seeds_main.txt > nodes_main.txt
+    curl https://bitcoin.sipa.be/asmap-filled.dat > asmap-filled.dat
```

Comment:
Should this filename be made consistent (one way or the other and .gitignore too) with `DEFAULT_ASMAP_FILENAME`?
```cpp
src/init.cpp:129:static const char* DEFAULT_ASMAP_FILENAME="ip_asn.map";
```

Reply:
I don't know, I think that use of asmap is completely unrelated. Using the same name would make sure if we were to use the one from the user's bitcoin directory. But we want to use a specific, deterministic asmap here so everyone gets the same result.

Reply:
Good point, agree.